dev/clint/src/clint/rules/do_not_disable.py

```diff
@@ -4,16 +4,34 @@
 
 
 class DoNotDisable(Rule):
-    DO_NOT_DISABLE = {"B006"}
+    DO_NOT_DISABLE = {
```

Comment:
```suggestion
    RULES = {
```

Let's rename this to `RULES`